accounts/auth.go

```diff
@@ -32,32 +34,62 @@ func Username() string {
 
 // # of Seconds since ...
 func ExpiryDate() int64 {
-	return time.Now().Add(2 * time.Hour).Unix()
+
+	// TODO: Implement some form of reissing the token, besides reregistering
+	return time.Now().Add(24 * 60 * time.Hour).Unix()
 }
 
 func SecretKey() []byte {
 	return []byte("RandomizeThisPlease")
 }
 
-func NewToken() *Token {
-	token := jwt.New(jwt.SigningMethodHS256)
-
-	claims := token.Claims.(jwt.MapClaims)
+func CreateToken() *Token {
```

Comment:
I like that everything token-related lives in its own module, but as a little code organization tip, I would make it into a some sort of service struct which can be configured and embedded into other structs if needed / implementation substituted. Something like:

```go
type Authenticator interface {
  IssueToken() (*Token, error)
  ValidateToken(Token) error
}

type jwtAuthenticator struct { } // implements Authenticator
```